components/core/Button.tsx

```diff
@@ -1,22 +1,66 @@
 import { chakra, Button as ChakraButton } from "@chakra-ui/react";
+import theme from "@/src/theme";
+
 
 export const Button = chakra(ChakraButton, {
   baseStyle: {
-    color: "red",
+    font: "Inter",
+    fontSize: "14px",
+    height: "auto",
+    borderRadius: "4px",
+    padding: "16px 24px"
   },
 });
 
 export const RedButton = chakra(ChakraButton, {
   baseStyle: {
     color: "white",
-    background: "#ff3030",
-    border: "solid",
+    background: "#BC002D",
+    border: "1px solid",
     width: "100%",
     margin: "0 auto",
     display: "block",
     _hover: {
-      bg: "#ffff",
-      color: "#ff3030",
+      bg: "white",
+      color: "#BC002D",
+    },
+  },
+});
+
+export const BlackButton = chakra(ChakraButton, {
+  baseStyle: {
+    font: "Inter-bold",
+    fontWeight: "400",
+    color: "white",
+    background: "black",
+    border: "1px solid black",
+    _hover: {
+      bg: theme.colors.brand.background_on,
     },
   },
 });
+
+export const WhiteButton = chakra(ChakraButton, {
+  baseStyle: {
+    font: "Inter-bold",
+    fontWeight: "400",
+    color: "black",
+    background: "white",
+    border: "1px solid black",
+    _hover: {
+      bg: theme.colors.brand.background_on,
```

Comment:
I think we should also change the button text too just to make it easier to read or maybe just invert the colors of the button